sdk/ml/azure-ai-ml/tests/automl_job/e2etests/test_automl_image_classification.py

```diff
@@ -117,6 +119,24 @@ def test_image_classification_multiclass_run(
             early_termination=BanditPolicy(evaluation_interval=2, slack_factor=0.2, delay_evaluation=6),
         )
 
+        # Configure component sweep job
+        image_classification_job_component_sweep = copy.deepcopy(image_classification_job)
+        image_classification_job_component_sweep.set_training_parameters(early_stopping=True, evaluation_frequency=1)
+        image_classification_job_component_sweep.extend_search_space(
+            [
+                SearchSpace(
+                    model_name=Choice(["microsoft/beit-base-patch16-224"]),
+                    learning_rate=Uniform(0.001, 0.01),
+                    number_of_epochs=Choice([15, 30]),
```

Comment:
any reason to use more than one epoch? number_of_epochs=Choice([1])

we need to run tests as fast as possible